### ISSUE TITLE
Test suite build failure

### ISSUE THREAD

**Issue by bergmark** [k0ral/opml-conduit#3]
When building stackage nightly

```
test/Arbitrary.hs:129:31:
    Overlapping instances for Arbitrary [NonEmpty (NonNull Text)]
      arising from a use of ‘arbitrary’
    Matching instances:
      instance [safe] Arbitrary a => Arbitrary [a]
        -- Defined in ‘Test.QuickCheck.Arbitrary’
      instance Arbitrary [NonEmpty (NonNull Text)]
        -- Defined at test/Arbitrary.hs:93:10
    In the second argument of ‘(<*>)’, namely ‘arbitrary’
    In the expression:
      OutlineBase <$> genNonNull <*> arbitrary <*> arbitrary
      <*> (Just <$> genTime)
      <*> arbitrary
    In an equation for ‘arbitrary’:
        arbitrary
          = OutlineBase <$> genNonNull <*> arbitrary <*> arbitrary
            <*> (Just <$> genTime)
            <*> arbitrary
```

**Comment by k0ral** [comment]
Can you please provide details about the build environment that led to the error ? I believe https://github.com/k0ral/opml-conduit/commit/2d882e30c23e6f01e74b2269b3d49f052641c00a should fix it, but I'd like to be sure before packaging a new release, and I've yet to reproduce it on my side (using ghc-7.10.3 resolver).

**Comment by bergmark** [comment]
it works!

**Comment by k0ral** [comment]
I've released 0.4.0.1 with the fix. Thanks for reporting !
